ff2mpv.js

```diff
@@ -1,12 +1,15 @@
-function onError(error) {
```

Comment:
Where did `onError` go?

Reply:
It is still there - it is just displayed weirdly by git...

Reply:
Its still there. It is just displayed a weird way in the diff.